tests/integration/test_render.py

```diff
@@ -194,3 +199,36 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_hidden(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index in range(len(zones)):
+            z = '#zone-{}'.format(index + 1)
+            for s in self.SIDES:
+                self.assertEqual(self._get_style(z, 'border{}Width'.format(s), True), '0px')
+                self.assertEqual(self._get_style(z, 'border{}Style'.format(s), True), 'none')
+
+    def test_borders_shown(self):
+        self.load_scenario(zone_borders=True)
+        zones = self._get_zones()
+        for index in range(len(zones)):
+            z = '#zone-{}'.format(index + 1)
+            for s in self.SIDES:
+                self.assertEqual(self._get_style(z, 'border{}Width'.format(s), True), '1px')
+                self.assertEqual(self._get_style(z, 'border{}Style'.format(s), True), 'dotted')
+                self.assertEqual(self._get_style(z, 'border{}Color'.format(s), True), Colors.DARK_GREY)
+
+    def test_labels_hidden(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for zone in zones:
+            zone_name = zone.find_element_by_css_selector('p.zone-name')
+            self.assertEqual(zone_name.get_attribute('class'), 'zone-name sr')
+
+    def test_labels_shown(self):
+        self.load_scenario(zone_labels=True)
+        zones = self._get_zones()
+        for zone in zones:
+            zone_name = zone.find_element_by_css_selector('p.zone-name')
+            self.assertEqual(zone_name.get_attribute('class'), 'zone-name')
```

Comment:
@agaylard This is great, thanks for adding this test and the one above! One small change that you could make here is the following: In this line

```python
zone_name = zone.find_element_by_css_selector('p.zone-name')
```

you are already asserting that `zone` contains a `<p>` element that has a class of `zone-name` (the call to `find_element_by_css_selector` would error out if that wasn't the case), so it's not necessary to check that again with `self.assertEqual(...)`. What you *do* want to assert, however, is that the element does *not* have a class of `sr`:

```python
self.assertNotIn('sr', zone_name.get_attribute('class'))
```

To be consistent could then also change the last line of the previous test to:

```python
self.assertIn('sr', zone_name.get_attribute('class'))
```